front/src/applications/stdcmV2/views/StdcmViewV2.tsx

```diff
@@ -3,18 +3,30 @@ import React, { useState } from 'react';
 import { useSelector } from 'react-redux';
 
 import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
 import { useOsrdConfSelectors } from 'common/osrdContext';
+import { useStoreDataForSpeedLimitByTagSelector } from 'common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector';
+import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
+import { getTrainScheduleV2Activated } from 'reducers/user/userSelectors';
 
 import StdcmConfig from '../components/StdcmConfig';
 import StdcmHeader from '../components/StdcmHeader';
+import StdcmResults from '../components/StdcmResults';
 import type { StdcmSimulationResult } from '../types';
 
 const StdcmViewV2 = () => {
   const { getScenarioID } = useOsrdConfSelectors();
   const scenarioID = useSelector(getScenarioID);
 
+  const trainScheduleV2Activated = useSelector(getTrainScheduleV2Activated);
+  const { speedLimitByTag } = useStoreDataForSpeedLimitByTagSelector();
+  const { rollingStock } = useStoreDataForRollingStockSelector();
+
   const [pathProperties, setPathProperties] = useState<ManageTrainSchedulePathProperties>();
```

Comment:
We will do it on the next PR, where we will handle multiples simulations

Reply:
Actually, you have to use the other pathPropperties, because it's the properties of the path the stdcm algo found, which may be different from the path found by the classic pathfinding algo with the input (the path found by the pathfinding algo may be occupied for instance, and the stdcm algo may return another path)

Reply:
you're right, fix